### ISSUE TITLE
Make the error reporting better

### ISSUE THREAD

**Issue by rarebreed** [RedHatQE/polarize#29]
Check your @TestDefinition for a missing testCaseID corresponding to projectID RedHatEnterpriseLinux7 in rhsm.cli.tests.GeneralTests.MyFirstPolarize_Test

Add this also:

```
To create a new testcase in Polarion and retrieve its testCaseID, set <enabled>true</enabled> under the <importer type="testcase"> section of your xml-config.xml file.
```
Also put this information in /tmp/polarize-errors.txt instead of /tmp/bad-functions.txt

**Comment by rarebreed** [comment]
The /tmp/polarize-warnings.txt now looks like this:

```
~/P/testpolarize ❯❯❯ cat /tmp/polarize-warnings.txt                                                                                                                                                       master ✱
In @TestDefinition for com.github.redhatqe.rhsm.testpolarize.TestReq.testBadProjectToTestCaseID, in project PLATTP, the testCaseID is an empty string even though the corresponding XML file is present and has ID = PLATTP-10202.
In @TestDefinition for com.github.redhatqe.rhsm.testpolarize.TestReq.testUpgrade, in project PLATTP, the testCaseID is an empty string even though the corresponding XML file is present and has ID = PLATTP-10068.
In @TestDefinition for com.github.redhatqe.rhsm.testpolarize.TestReq.testError, in project PLATTP, the testCaseID is an empty string even though the corresponding XML file is present and has ID = PLATTP-10203.
In @TestDefinition for com.github.redhatqe.rhsm.testpolarize.TestReq.testTestDefinition, in project PLATTP, the testCaseID is an empty string even though the corresponding XML file is present and has ID = PLATTP-10547.
In @TestDefinition for com.github.redhatqe.rhsm.testpolarize.TestPolarize.testMethod, in project PLATTP, the testCaseID is an empty string even though the corresponding XML file is present and has ID = PLATTP-10069.
In @TestDefinition for com.github.redhatqe.rhsm.testpolarize.TestPolarize.yetAnotherTestMethod, in project PLATTP, the testCaseID is an empty string even though the corresponding XML file is present and has ID = PLATTP-10549.
```

**Comment by rarebreed** [comment]
Fixed in c108141f433cd9afe6216bc19cc146e1010da5f8
